frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/field-utils.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+
+export const SidebarInputWrapper: React.FC = ({ children }) => {
+  return <div style={{ width: 'calc(100% - 28px)' }}>{children}</div>;
```

Comment:
I'm not happy with this -- open to ideas on how I can keep the width of the inputs the same throughout the sidebar (the string[] guy comes with a "-" sign on the right side and that's 28px wide).